astrology.lic

```diff
@@ -18,15 +18,25 @@ class Astrology
     @ap_training = settings.astral_plane_training['train_in_ap']
     @ap_source = settings.astral_plane_training['train_source']
     @ap_destination = settings.astral_plane_training['train_destination']
-
+    @pools = {
```

Comment:
Does this need to be an instance var, or could it be passed into and out of methods?